.changeset/weak-taxis-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+Fixed handling of attachments in Omnichannel email transcripts. Earlier attachments were being skipped and were being shown as empty space, now it should render the image attachments and should show relevant error message for unsupported attachments.
```

Comment:
I believe we didn't fix a problem here, we added the support for it instead, right?

Reply:
Same, PR title says `feat`...